tagged.cabal

```diff
@@ -14,26 +14,29 @@ description:    Haskell 98 phantom types to avoid unsafely passing dummy argumen
 build-type:     Simple
 cabal-version:  >= 1.10
 extra-source-files: .hlint.yaml CHANGELOG.markdown README.markdown
-tested-with:   GHC == 7.0.4
-             , GHC == 7.2.2
-             , GHC == 7.4.2
-             , GHC == 7.6.3
-             , GHC == 7.8.4
-             , GHC == 7.10.3
-             , GHC == 8.0.2
-             , GHC == 8.2.2
-             , GHC == 8.4.4
-             , GHC == 8.6.5
-             , GHC == 8.8.4
-             , GHC == 8.10.7
-             , GHC == 9.0.2
-             , GHC == 9.2.8
-             , GHC == 9.4.5
-             , GHC == 9.6.2
+tested-with:
+  -- GHC == 7.0.4
+  -- GHC == 7.2.2
+  -- GHC == 7.4.2
+  -- GHC == 7.6.3
+  -- GHC == 7.8.4
```

Comment:
Why does this change comment out these versions? (Perhaps there is a motivation here, but it's not explained in the commit message.)

Reply:
My patched version of `haskell-ci` defaults to `distribution: focal` (20.04) and `ghcup` as installation method, thus, does not support these older ghc versions out of the box.

If you wish to keep them tested by CI, I can switch to `distribution: bionic` (18.04) and restore them.

Reply:
I restored CI for GHC 7

Reply:
Ah, I see. I'm content to leave it on Ubuntu 18.04 for now, but I suppose we will need to address this at some point or another. In any case, that's best left for a different PR.